modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -357,9 +357,40 @@ class ResizeSubgraph2 : public ExtractScalesSubgraph
     }
 };
 
+class BatchNormalizationSubgraph : public Subgraph
+{
+public:
+    BatchNormalizationSubgraph()
+    {
+        int input = addNodeToMatch("");
+        int data1 = addNodeToMatch("Constant");
+        int data2 = addNodeToMatch("Constant");
+        int data3 = addNodeToMatch("Constant");
+        int data4 = addNodeToMatch("Constant");
+        int shape1 = addNodeToMatch("Constant");
+        int reshape1 = addNodeToMatch("Reshape", data1, shape1);
+        int shape2 = addNodeToMatch("Constant");
+        int reshape2 = addNodeToMatch("Reshape", data2, shape2);
+        int shape3 = addNodeToMatch("Constant");
+        int reshape3 = addNodeToMatch("Reshape", data3, shape3);
+        int shape4 = addNodeToMatch("Constant");
+        int reshape4 = addNodeToMatch("Reshape", data4, shape4);
+        int sqrt = addNodeToMatch("Sqrt", reshape3);
+        int A = addNodeToMatch("Constant");
+        int div = addNodeToMatch("Div", A, sqrt);
```

Comment:
please replace `int sqrt` and `int div` to something like `sqrtNode` and `divNode` to avoid conflicts with std namespace.

Reply:
sure.